DefaultNotificationBody.android.js

```diff
@@ -42,8 +62,8 @@ export default class DefaultNotificationBody extends React.Component {
       onClose,
     } = this.props;
 
-    return(
-      <GestureRecognizer onSwipe={this._onSwipe.bind(this)} style={styles.container}>
+    return (
+      <GestureRecognizer onSwipe={direction => this.onSwipe(direction)} style={styles.container}>
```

Comment:
Here, can you bind `this` in the constructor please:
```
constructor() {
  super();
  
  this.onSwipe = this.onSwipe.bind(this);
}
```
Then in the `onSwipe` prop do `onSwipe={this.onSwipe}`.